### PR TITLE
[Release] Add Arc Appliance Extension 0.2.15

### PR DESCRIPTION
**Pull request by pkoushik**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [X] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`. 

WI: https://msazure.visualstudio.com/AzureArcPlatform/_queries/query/22f18f13-1727-4d68-9c62-3b225db7e791/

### CODE REVIEW

**Comment by pkoushik** [comment]
Please reach out to me before merging.

**Comment by yonzhan** [comment]
Arc Appliance

**Comment by pkoushik** [comment]
@kairu-ms feel free to merge this when you can

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**
